src/container/BooksList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+
+const BooksList = ({ bookId, title, category }) => (
+  <tr>
+    <th>BookId</th>
+    <th>Title</th>
+    <th>Category</th>
+    <Book bookId={bookId} title={title} category={category} />
+  </tr>
+);
+
+BooksList.propTypes = {
+  bookId: PropTypes.number,
+  title: PropTypes.string,
+  category: PropTypes.string,
+};
+
+BooksList.defaultProps = {
+  bookId: Math.random(),
+  title: '',
+  category: '',
+};
```

Comment:
- [ ] <b>REQUIRED:</b> Since we are expecting only the list of the books in the props, we don't have to define those `BookId, title or category` in default props, we just have to define `books` with an empty array as a value. just like this:
```Javascript
BooksList.defaultProps = {
  books: [],
};
```